README.rst

```diff
@@ -103,6 +103,7 @@ Options
 -  ``content_security_policy_report_uri``, default ``None``, a string
    indicating the report URI used for `CSP violation reports
    <https://developer.mozilla.org/en-US/docs/Web/Security/CSP/Using_CSP_violation_reports>`_
+- ``support_x_header``, default ``True``, Whether to add X-CSP header
```

Comment:
Let's call this `legacy_content_security_policy_header`. Yeah, it's verbose, but it fits with the rest of the options and makes it immediately obvious what this does. Might be useful to link to the page that describes the deprecated header.

Reply:
done it in commit b688897 (also i linted the test_talisman file)